### PR TITLE
Removed EBI layer on MSA-related pipelines

### PR DESCRIPTION
**Pull request by JAlvarezJarreta**
Ticket [ENSCOMPARASW-2486](https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-2486) (part 3/7).

### CODE REVIEW

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Legacy/EPO_pt2_conf.pm [hunk 1]
```diff
@@ -25,26 +25,18 @@ Bio::EnsEMBL::Compara::PipeConfig::EPO_pt2_conf
...
-
-	'ensembl_cvs_root_dir' - the path to the compara/hive/ensembl GIT checkouts - set as an environment variable in your shell
-	'compara_anchor_db' - database containing the anchor sequences (entered in the anchor_sequence table)
-	'master_db' - location of your master db containing relevant info in the genome_db, dnafrag, species_set, method_link* tables
-        The dummy values - you should not need to change these unless they clash with pre-existing values associated with the pairwise alignments you are going to use
-
     #4. Run init_pipeline.pl script:
-        Using command line arguments:
-        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EPO_pt2_conf.pm
+        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EPO_pt2_conf -host mysql-ens-compara-prod-X -port XXXX \
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Legacy/EPO_pt2_conf.pm]:
```suggestion
        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Legacy::EPO_pt2_conf -host mysql-ens-compara-prod-X -port XXXX \
```

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Legacy/EPO_pt3_conf.pm [hunk 2]
```diff
@@ -47,7 +47,9 @@ Bio::EnsEMBL::Compara::PipeConfig::EPO_pt3_conf
     #3. make sure that all default_options are set correctly
 
     #4. Run init_pipeline.pl script:
-        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EPO_pt3_conf -password <your_password> -mlss_id <your_current_epo_mlss_id> -species_set_name <the name of the species set> -compara_mapped_anchor_db <db name from epo_pt2 pipeline> -master_db <>
+        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EPO_pt3_conf -host mysql-ens-compara-prod-X -port XXXX \
```
Comment (carlacummins) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Legacy/EPO_pt3_conf.pm]:
```suggestion
        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Legacy::EPO_pt3_conf -host mysql-ens-compara-prod-X -port XXXX \
```

**Comment by carlacummins** [comment]
Github reporting conflicts in this file: `modules/Bio/EnsEMBL/Compara/PipeConfig/EPO_pt3_conf.pm`

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Legacy/EPO_pt2_conf.pm [hunk 3]
```diff
@@ -19,32 +19,24 @@ limitations under the License.
...
+            -division $COMPARA_DIV -species_set_name <species_set_name> -mlss_id <curr_epo_mlss_id>
 
     #5. Run the "beekeeper.pl ... -sync" and then " -loop" command suggested by init_pipeline.pl
 
     #6. Fix the code when it crashes
 
 =head1 DESCRIPTION  
 
-    This configuaration file gives defaults for mapping (using exonerate at the moment) anchors to a set of target genomes (dumped text files)
+    This configuaration file gives defaults for mapping (using exonerate at the
```
Comment (muffato) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Legacy/EPO_pt2_conf.pm]:
```suggestion
    This configuration file gives defaults for mapping (using exonerate at the
```

**Pull request merged**